File: pinyougou-sellergoods-interface/src/main/java/com/pinyougou/sellergoods/service/BrandService.java
package com.pinyougou.sellergoods.service;



import com.pinyougou.pojo.TbBrand;

import entity.PageResult;
import entity.Result;

import java.util.List;
import java.util.Map;

public interface BrandService {
    /**
     * 分页,条件查询
     */
	PageResult search(TbBrand tbBrand,int pagenum,int pagesize);
	
	/**
	 * 增加
	 * @param tbBrand
	 * @return
	 */
	void add(TbBrand tbBrand);
	
	/**
	 * 修改(返回的到页面数据)
	 * 
	 */
	TbBrand findOne(long id);
	/**
	 * 保存修改的数据
	 */
	
	void save(TbBrand tbBrand);
	
	/**
	 * 删除
	 */
	
	void delete(long[] ids);

	/**
	 * 下拉品牌列表
	 */
	List<Map> selectOptionList();
}

File: pinyougou-manager-web/src/main/webapp/js/service/brandService.js
    //品牌服务
      app.service("brandService",function($http){
    	  //分页,条件查询
    	  this.search=function(page,size,searchEntity){
    		 return $http.post("../brand/search.do?pagenum="+page+"&pagesize="+size,searchEntity)
    	  }
    	  //增加
    	  this.add=function(entity){
    		  
    		  return $http.post("../brand/add.do",entity)
    		  
    	  }
    	  this.update=function(entity){
    		  
    		  return $http.post("../brand/update.do",entity)
    	  }
    	  
		 this.findOne=function(id){
    		  
    		  return $http.get("../brand/findOne.do?id="+id)
    	  }
		 
		 this.del=function(selectids){
		    return $http.post("../brand/delete.do",selectids)
		 }
		 this.selectOptionList=function () {
			 return $http.post("../brand/selectOptionList.do")
         }
      });